other/EXAMPLES.md

```diff
@@ -1,3 +1,55 @@
 # Examples
 
-There aren't any examples yet! Want to add one? See `CONTRIBUTING.md`
+Want to add an example? See `CONTRIBUTING.md`
+
+## Working with other plugins
```

Comment:
@kentcdodds Awesome, this plus the `babelRcPath` that was added takes care of #1